src/components/_shared/table.js

```diff
@@ -35,4 +36,12 @@ const Table = ({ array, result, section }) => {
   );
 };
 
+Table.propTypes = {
+  array: PropTypes.array,
+  result: PropTypes.string,
+  section: PropTypes.string,
```

Comment:
`Section` should be `isGamesStats` which would be a boolean. This would required code change. Similar would need this change in `_shared/title/index.js`